.github/workflows/test-documentation-build.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Traits documentation build
+
+on:
+  pull_request
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.8]
```

Comment:
Not sure why we are trying to build docs on both 3.6 and 3.8.

Reply:
Again, because there may be Python version dependencies. For example, there may be more recent versions of Sphinx available for Python 3.8. This is a compromise between running on all possible combinations and running on just one.